es73x/src/main/scala/tech/beshu/ror/es/providers/EsAuditSink.scala

```diff
@@ -40,7 +40,7 @@ class EsAuditSink(client: Client) extends AuditSink with Logging {
       .build
 
   override def submit(indexName: String, documentId: String, jsonRecord: String): Unit = {
-    bulkProcessor.add(new IndexRequest(indexName, "ror_audit_evt", documentId).source(jsonRecord, XContentType.JSON))
+    bulkProcessor.add(new IndexRequest(indexName).source(jsonRecord, XContentType.JSON))
```

Comment:
I know it's deprecated, but IMO we should leave it as it was before. Types will be removed in ES 8.x, so we will have to prepare for it after all (https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html). But IMO we have to do it wisely, with proper tests. Let's abandon the idea to change it now.